### PR TITLE
EXID.TYPE for BillionGraves.com and WikiTree identifiers

### LINKED ISSUE #539: EXID.TYPE for BillionGraves.com and WikiTree identifiers
Identifiers for [BillionGraves.com](https://www.billiongraves.com/) and [WikiTrees](https://www.wikitree.com/) are included in some GEDCOM files using extension tags (`_BGRAVE` `_BGLINK` `_WIKITREE` `_WIKILINK`).

This request is for the following EXID types to be registered to support greater compatibility (as was done in #261).

Description | Type | Example | Reference
-- | -- | -- | --
BillionGraves Cemetery ID | https://www.billiongraves.com/cemetery/name/ | George Washington Memorial Park [https://billiongraves.com/cemetery/name/68270](https://billiongraves.com/cemetery/name/68270) | https://billiongraves.com/search/cemetery
BillionGraves Grave ID | https://www.billiongraves.com/grave/name/ | George Washington [https://billiongraves.com/grave/name/361654](https://billiongraves.com/grave/name/361654) | https://billiongraves.com/search
WikiTree Person ID | https://www.wikitree.com/wiki/ | George Washington [https://www.wikitree.com/wiki/Washington-2114](https://www.wikitree.com/wiki/Washington-2114) | https://www.wikitree.com/wiki/Help:WikiTree_ID

### PR DESCRIPTION
**Pull request by elyoh**
Fixes #539.

See #262 for similar past request.

### CODE REVIEW

File: exid-types.json [hunk 1]
```diff
@@ -81,4 +81,28 @@
     "change-controller": "FindAGrave.com",
     "reference": "https://support.findagrave.com/s/article/Cemetery-Search"
   }
+  {
+    "label": "BillionGraves Grave ID",
+    "type": "https://www.billiongraves.com/grave",
```
Comment (tychonievich) [inline exid-types.json:86]:
```suggestion
    "type": "https://www.billiongraves.com/grave/",
```
Addition to make the type + id = URL, as recommended in the spec:

> If the authority maintains stable URLs for each identifier it issues, it is recommended that the [TYPE](https://gedcom.io/specifications/FamilySearchGEDCOMv7.html#TYPE) payload be selected such that appending the [EXID](https://gedcom.io/specifications/FamilySearchGEDCOMv7.html#EXID) payload to it yields that URL.

File: exid-types.json [hunk 2]
```diff
@@ -81,4 +81,28 @@
...
+    "description": "BillionGraves Grave ID",
+    "contact": "GEDCOM@familysearch.org",
+    "change-controller": "BillionGraves.com",
+    "reference": "https://billiongraves.com/search"
+  },
+  {
+    "label": "BillionGraves Cemetery ID",
+    "type": "https://www.billiongraves.com/cemetery",
```
Comment (tychonievich) [inline exid-types.json:94]:
```suggestion
    "type": "https://www.billiongraves.com/cemetery/",
```

File: exid-types.json [hunk 3]
```diff
@@ -81,4 +81,28 @@
...
+    "description": "BillionGraves Cemetery ID",
+    "contact": "GEDCOM@familysearch.org",
+    "change-controller": "BillionGraves.com",
+    "reference": "https://billiongraves.com/search/cemetery"
+  }
+  {
+    "label": "WikiTree Person ID",
+    "type": "https://www.wikitree.com/wiki",
```
Comment (tychonievich) [inline exid-types.json:102]:
```suggestion
    "type": "https://www.wikitree.com/wiki/",
```

File: exid-types.json [hunk 4]
```diff
@@ -81,4 +81,28 @@
...
+    "description": "BillionGraves Cemetery ID",
+    "contact": "GEDCOM@familysearch.org",
+    "change-controller": "BillionGraves.com",
+    "reference": "https://billiongraves.com/search/cemetery"
+  }
+  {
+    "label": "WikiTree Person ID",
+    "type": "https://www.wikitree.com/wiki/",
+    "description": "WikiTree Person ID",
+    "contact": "GEDCOM@familysearch.org",
```
Comment (dthaler) [inline exid-types.json:104]: @clarkegj Can this be updated in the future to have a wikitree contact here?

**Review by dthaler — APPROVED** [review]
Approved by GEDCOM Steering Committee

File: exid-types.json (hunk 1 above)
Comment (elyoh) [reply exid-types.json]: In this case the type should be https://www.billiongraves.com/grave/name/ as appending the EXID payload to that gives the resource.

File: exid-types.json (hunk 2 above)
Comment (elyoh) [reply exid-types.json]: Likewise here, it would be https://www.billiongraves.com/cemetery/name/

**Pull request merged**